### PR TITLE
Add Slack Chat link to docs and PR template

### PR DESCRIPTION
**Pull request by Crissman**
Add Slack Chat and Chainer forums link to docs index
Add Slack Chat to PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ Please double-check the following.
   - Did you write sufficient test code?
   - Did you write sufficient documentation?
 - Also take a look at [our compatibility policy](http://docs.chainer.org/en/stable/compatibility.html).
+
+Support is available on our [Slack Chat](https://bit.ly/chainer-slack).
```
Comment (niboshi) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is the pull request template. How about writing in the issue template instead?

File: docs/source/index.rst [hunk 2]
```diff
@@ -27,10 +27,16 @@ This is the `Chainer <https://chainer.org>`_ documentation.
...
 Indices and tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Other Support
```
Comment (niboshi) [inline docs/source/index.rst]: How about making a section "Community" and introducing other resources, including awesome-chainer?

Comment (niboshi) [inline docs/source/index.rst]: Also it's better to fix the stylesheet to make it easy to distinguish between internal and external links.

`<a>` elements in the navigation menu have `internal` and `external` classes respectively.

**Comment by kmaehashi** [comment]
Let's use a Slack sharing link.

* http://bit.ly/join-chainer-slack
* http://bit.ly/join-chainer-jp-slack

Could you please replace the links in README, too?

**Comment by Crissman** [comment]
Done, and added arrows for outgoing links.

screen shot 2018-01-18 at 10 17 21 am

**Comment by niboshi** [comment]
jenkins, test this please

**Comment by niboshi** [comment]
jenkins, test this please

**Comment by niboshi** [comment]
LGTM!

**Pull request merged**